recipes/pep8-naming/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.3.3" %}
+
+package:
+  name: pep8-naming
+  version: {{ version }}
+
+source:
+  fn: pep8-naming-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/p/pep8-naming/pep8-naming-{{ version }}.tar.gz
+  md5: 566f05660200993191312d51cd5a6cc9
+
+build:
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - flake8
+
+test:
+  commands:
+    - "flake8 --version | grep naming:"
```

Comment:
> I'll defer to your view here, though, if you think it serves a purpose.

The only purpose I can come up with is if your pep8 check changes or adds a "naming:" at some point which doesn't relate to this. It is pretty unlikely IMHO.

Question: Does grep just work on our AppVeyor builds? Seems it does! :+1: